prometheus/histogram.go

```diff
@@ -308,23 +308,23 @@ func NewHistogramVec(opts HistogramOpts, labelNames []string) *HistogramVec {
 }
 
 // GetMetricWithLabelValues replaces the method of the same name in
-// MetricVec. The difference is that this method returns a Histogram and not a
+// MetricVec. The difference is that this method returns an Observer and not a
 // Metric so that no type conversion is required.
```

Comment:
These comments don't apply to all cases anymore, right? Using `Write()` method will now require a type conversion.